torch/csrc/jit/runtime/profiling_graph_executor_impl.h

```diff
@@ -4,6 +4,11 @@
 namespace torch {
 namespace jit {
 
+TORCH_API Node* createFallbackGraph(
```

Comment:
This is an unused API atm - do you mind adding a test when you plan to land this